src/explore-education-statistics-admin/src/services/applicationInsightsService.ts

```diff
@@ -0,0 +1,25 @@
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import client from '@admin/services/util/service';
+
+const service = {
+  GetInsightsKey(): Promise<string> {
+    return client.get<string>('/configuration/application-insights');
+  },
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const initApplicationInsights = async () => {
+  const key = await service.GetInsightsKey();
+  if (key) {
+    const appInsights = new ApplicationInsights({
+      config: {
+        instrumentationKey: key,
+        enableAutoRouteTracking: true,
+        autoTrackPageVisitTime: true,
+      },
+    });
+    appInsights.loadAppInsights();
+
+    console.log('Application Insights initialised');
```

Comment:
Remove console.log

Reply:
@benoutram I put this in because its the same on the frontend and just clarifies everything has started up